--- a/ibv-conduit/gasnet_core.c.orig	2018-04-12 16:34:54.942366000 -0400
+++ b/ibv-conduit/gasnet_core.c	2018-04-12 16:36:23.917532000 -0400
@@ -77,7 +77,7 @@
 
 /* HW level retry knobs */
 int gasnetc_qp_timeout, gasnetc_qp_retry_count;
-#define GASNETC_DEFAULT_QP_TIMEOUT		18	/* about 1s */
+#define GASNETC_DEFAULT_QP_TIMEOUT		31	/* about 1s */
 #define GASNETC_DEFAULT_QP_RETRY_COUNT		7
 
 #ifndef MT_MELLANOX_IEEE_VENDOR_ID
